### DISCUSSION TITLE
Recent file list being generated in Windows taskbar

### DISCUSSION THREAD

**Discussion by Mister-Curious** [smplayer-dev/smplayer#896] [Q&A] [answered]
I recently noticed when right clicking on the SMPlayer item in the Windows tasksbar that a list of recent files is generated:

![image](https://github.com/smplayer-dev/smplayer/assets/81539091/f2a2c9e3-4eff-4983-b0dd-035df8a0ee6f)

This happens despite that I have set recent files to be disabled in Options--->Preferences--->Interface--->Privacy (with Max recent items set to a value of 0):

![image](https://github.com/smplayer-dev/smplayer/assets/81539091/fa108c76-b9dd-45d9-9d04-9625003a090e)

This seems likes a bug unless I am missing something. 

Using Version: 22.2.0 (revision 10060) (64-bit)
Using Qt 5.15.2 (compiled with Qt 5.15.2)
Using MPV 0.34.0

On Windows 10 Pro 64 bit

**Comment by smplayer-dev** [comment, marked as answer]
That list isn't created by smplayer but by Windows. 
You can turn it off:
https://www.windowscentral.com/how-disable-recent-files-and-locations-jump-lists-windows-10
